### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by ale-kacperczyk**
# Description

- Removed the info about versioning (it's visible in docs)
- Updated the # of customers
- A couple of wording changes regarding the rename of the Platform
- Removed some instructions that are only needed for the internal users

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 
 Hello 👋!
 
-This is the for all of the [LiveChat Platform documentation](https://developers.livechat.com/docs/).
+This is the repository for the [Platform documentation](https://developers.livechat.com/docs/).
 Our documents are written in the [MDX format](https://mdxjs.com/), so pay attention to our custom tags when contributing to the docs.
 
 ## Who we are
 
 Behind LiveChat, Inc. there is a team of [passionate people](https://www.livechat.com/team/) building online customer service software with online chat, help desk software, chatbot, and web analytics capabilities.
```
Comment (kamilatokarska) [inline README.md:12]: Are we leaving `LiveChat` or changing to `Text` like in Docs?

File: README.md [hunk 2]
```diff
@@ -30,84 +30,25 @@ We require node version 14.x.
...
 
 1. Run `npm run build`
 2. Run `npm start`
 
-## Feature branch / Deploy Preview
+## Feature branch / Deploy preview
 
-In order to test your work before merging to master, **Netlify** provides **Deploy Preview**. When raising a PR, github will provide **Netlify** notifications with the **Deploy Preview** url.
-
-## Versioning
```
Comment (kamilatokarska) [inline README.md:44]: Is there a particular reason why we remove it completely? Maybe we could link the docs with versioning here? 🤔

Comment (ale-kacperczyk) [reply README.md:44]: When I think about it, I don't know why someone might need to know the versioning of some of our APIs while reading README. The versioning is mentioned and explained several times in the docs, and the README is mostly about what this repo is, what the PLatform is, what Developer Program is, what you can build, how you can run the repository, so instructions on how to do things with the repository

File: README.md (hunk 1 above)
Comment (kamilatokarska) [reply README.md:12]: Okay, I wasn't sure when we implement the changes to the naming, so I preferred to ask😉

File: README.md (hunk 2 above)
Comment (kamilatokarska) [reply README.md:44]: Okay, that makes a total sense😊

**Review by kamilatokarska — APPROVED** [review]
😊

**Pull request merged**